docs/src/main/paradox/index.md

```diff
@@ -15,10 +15,12 @@ connection.
 Based on a protobuf service definition, akka-grpc can generate:
 
 * Model classes (using plain protoc for Java or scalapb for Scala)
-* The API (as an interface for Java or a trait for Scala), expressed in Akka Streams `Source`s
+* The API trait or interface, expressed in Akka Streams `Source`s
```

Comment:
`@scala[trair or]` java people react with panic sometimes to scala wording

Reply:
🙀